src/server.cpp

```diff
@@ -443,7 +443,8 @@ void Server::start(Address bind_addr)
 	m_thread->start();
 
 	// ASCII art for the win!
-	actionstream
+	// http://patorjk.com/software/taag/#p=display&h=1&f=Small%20Shadow&t=minetest
```

Comment:
IMHO, this is the correct link:
http://patorjk.com/software/taag/#p=display&h=1&f=Graffiti&t=minetest